src/function/pragma/pragma_queries.cpp

```diff
@@ -4,35 +4,40 @@
 
 namespace duckdb {
 
-string pragma_table_info(ClientContext &context, vector<Value> parameters) {
+string pragma_table_info(ClientContext &context, vector<Value> parameters,
+                         unordered_map<string, Value> named_parameters) {
 	return StringUtil::Format("SELECT * FROM pragma_table_info('%s')", parameters[0].ToString());
 }
 
-string pragma_show_tables(ClientContext &context, vector<Value> parameters) {
+string pragma_show_tables(ClientContext &context, vector<Value> parameters,
+                          unordered_map<string, Value> named_parameters) {
 	return "SELECT name FROM sqlite_master() ORDER BY name";
 }
 
-string pragma_database_list(ClientContext &context, vector<Value> parameters) {
+string pragma_database_list(ClientContext &context, vector<Value> parameters,
+                            unordered_map<string, Value> named_parameters) {
 	return "SELECT * FROM pragma_database_list() ORDER BY 1";
 }
 
-string pragma_collations(ClientContext &context, vector<Value> parameters) {
+string pragma_collations(ClientContext &context, vector<Value> parameters,
+                         unordered_map<string, Value> named_parameters) {
 	return "SELECT * FROM pragma_collations() ORDER BY 1";
 }
 
-string pragma_show(ClientContext &context, vector<Value> parameters) {
+string pragma_show(ClientContext &context, vector<Value> parameters, unordered_map<string, Value> named_parameters) {
```

Comment:
Maybe an idea to pack the parameters into a struct, e.g.

```cpp
struct FunctionParameters {
   vector<Value> values;
   unordered_map<string, Value> named_parameters;
};
```

That can also be used in the table functions.

Reply:
Good idea, this makes it look more clean